### PR TITLE
Jmb145

### PR DESCRIPTION
**Pull request by jordieburton3**
Writes the teams, meets, and participates values into the mysql database.

### CODE REVIEW

File: helpers/db.js [hunk 1]
```diff
@@ -77,6 +77,11 @@ const drop = (tables, done) => {
   );
 };
 
+connect(MODE_TEST, () => {
```
Comment (zacharyfmarion) [inline helpers/db.js]: This doesn't make sense...you should be wrapping all the code you call in this, not calling it in the helper file and no-oping the callback

File: package.json [hunk 2]
```diff
@@ -5,7 +5,8 @@
   "main": "index.js",
   "scripts": {
     "start": "nodemon index.js --exec babel-node --presets  es2015,stage-2",
-    "scrape": "babel-node helpers/scraper.js --presets es2015,stage-2",
+    "scrape": "babel-node scraper.js --presets es2015,stage-2",
+    "db": "babel-node helpers/db.js --presets es2015,stage-2",
```
Comment (zacharyfmarion) [inline package.json]: ?

File: scraper.js [hunk 3]
```diff
@@ -0,0 +1,494 @@
+import request from "request";
+import cheerio from "cheerio";
+import db from "./helpers/db";
+
+let urls = [];
```
Comment (zacharyfmarion) [inline scraper.js]: Not sure why all of this code isn't just wrapped in db.connect

**Review by zacharyfmarion — CHANGES_REQUESTED** [review]

File: helpers/db.js (hunk 1 above)
Comment (jordieburton3) [reply helpers/db.js]: tbh I don't even know why I put that there.

File: package.json (hunk 2 above)
Comment (jordieburton3) [reply package.json]: Yeah idk man, I'll get rid of it

File: scraper.js (hunk 3 above)
Comment (jordieburton3) [reply scraper.js]: Would it be better to have everything in a db.connect? what would be the benefit? how would one do such a thing?

**Pull request merged**